.claude/scripts/poll-pr.sh

```
@@ -1,191 +1,597 @@
-  echo ""
+}
+
+# Fetch all CodeRabbit review threads via GraphQL (includes resolved/outdated status)
+fetch_review_threads() {
+  local owner="${REPO%%/*}" rname="${REPO##*/}"
+  gh api graphql \
+    -f query='query($owner:String!,$repo:String!,$pr:Int!){
+      repository(owner:$owner,name:$repo){pullRequest(number:$pr){reviewThreads(first:100){nodes{
+        id isResolved isOutdated
+        comments(first:3){nodes{databaseId author{login __typename} body path line originalLine}}
+      }}}}}'  \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)